packages/web/src/js/FileUploader.ts

```diff
@@ -124,12 +144,22 @@ class FileUploader extends BaseComponent {
     }
 
     if (!isTypeSupported) {
-      throw new Error(
-        `The file "${file.name}" is not supported. Please ensure you are uploading a supported file format.`,
-      );
+      throw new Error(`"${file.name}": ${errorMessages.errorFileNotSupported}`);
```

Comment:
is it ok we don't throw our own error event too here? I don't know :)

Reply:
it is ok to throw here